Upstream patch won't apply cleanly because they moved the firmware line into it's own 50-firmware.rules
file in git. This is the modified copy for 198.

http://bugs.gentoo.org/show_bug.cgi?id=462346

http://cgit.freedesktop.org/systemd/systemd/patch/?id=bbb7f2ae5035105575365750592caa87213d7072

--- rules/50-udev-default.rules
+++ rules/50-udev-default.rules
@@ -1,5 +1,13 @@
 # do not edit this file, it will be overwritten on update
 
+SUBSYSTEM=="rtc", ATTR{hctosys}=="1", SYMLINK+="rtc"
+SUBSYSTEM=="virtio-ports", KERNEL=="vport*", ATTR{name}=="?*", SYMLINK+="virtio-ports/$attr{name}"
+
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", IMPORT{builtin}="usb_id", IMPORT{builtin}="hwdb --subsystem=usb"
+SUBSYSTEM=="input", ENV{ID_INPUT}=="", IMPORT{builtin}="input_id"
+SUBSYSTEM=="firmware", ACTION=="add", RUN{builtin}="firmware"
+ENV{MODALIAS}!="", IMPORT{builtin}="hwdb --subsystem=$env{SUBSYSTEM}"
+
 ACTION!="add", GOTO="default_permissions_end"
 
 SUBSYSTEM=="tty", KERNEL=="ptmx", GROUP="tty", MODE="0666"
@@ -8,6 +16,8 @@
 SUBSYSTEM=="vc", KERNEL=="vcs*|vcsa*", GROUP="tty"
 KERNEL=="tty[A-Z]*[0-9]|pppox[0-9]*|ircomm[0-9]*|noz[0-9]*|rfcomm[0-9]*", GROUP="dialout"
 
+SUBSYSTEM=="rtc", ATTR{hctosys}=="1", MODE="0644"
+
 SUBSYSTEM=="mem", KERNEL=="mem|kmem|port", GROUP="kmem", MODE="0640"
 
 SUBSYSTEM=="input", KERNEL=="mouse*|mice|event*", MODE="0640"
@@ -59,10 +69,3 @@
 KERNEL=="fuse", MODE="0666", OPTIONS+="static_node=fuse"
 
 LABEL="default_permissions_end"
-SUBSYSTEM=="rtc", ATTR{hctosys}=="1", MODE="0644", SYMLINK+="rtc"
-SUBSYSTEM=="virtio-ports", KERNEL=="vport*", ATTR{name}=="?*", SYMLINK+="virtio-ports/$attr{name}"
-
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", IMPORT{builtin}="usb_id", IMPORT{builtin}="hwdb --subsystem=usb"
-SUBSYSTEM=="input", ENV{ID_INPUT}=="", IMPORT{builtin}="input_id"
-SUBSYSTEM=="firmware", ACTION=="add", IMPORT{builtin}="firmware"
-ENV{MODALIAS}!="", IMPORT{builtin}="hwdb --subsystem=$env{SUBSYSTEM}"
